### DISCUSSION TITLE
[JavaScript|Python] DateTimeV2 support for it-it

### DISCUSSION THREAD

**Discussion by bobokids** [microsoft/Recognizers-Text#2234] [General]
**Describe the bug**
Please refer to https://github.com/microsoft/botframework-skills/issues/282. it-it does not support datetimev2 entity.

**Platform:**
 - Platform: JavaScript, Python

**Comment by tellarin** [comment]
@bobokids, DateTimeV2 is already supported in .NET. 
Would yo be able to help porting it to JavaScript/Python?
